adal/devApps/AdalAndroidTestApp/MainActivity.cs

```diff
@@ -134,34 +133,5 @@ private async void clearCacheButton_Click(object sender, EventArgs e)
                 this.accessTokenTextView.Text = "Cache cleared";
             });
         }
-
-        private async void conditionalAccessButton_Click(object sender, EventArgs e)
```

Comment:
>rivate async void conditionalAccessButton_Click(object sender, EventArgs e) [](start = 9, length = 75)

Are we sure we do not need dev samples for CA? to make testing CA as easy as possible? Do we believe the PM sample is enough also for debugging?

Reply:
I'll sync with @jmprieur on that. i don't think this particular test is of value because the claims are mocked, so it's just using the claims AT overload. We could integrate a true CA e2e test. I'll investigate.

---
In reply to: [211379932](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1211#discussion_r211379932) [](ancestors = 211379932)